src/main/java/com/github/steveice10/mc/auth/service/MsaAuthenticationService.java

```diff
@@ -295,7 +343,7 @@ public String toString() {
 
         protected MsCodeRequest(String clientId) {
             this.client_id = clientId;
-            this.scope = "XboxLive.signin";
+            this.scope = "XboxLive.signin offline_access";
```

Comment:
Does this break existing functionality in any way?

Reply:
Nope, doesn't break anything. Only really noticable change is what appears in the permissions page: ![new permission with `offline_access`](https://ass.rip/Y7Pygn6y5uhu/direct)

Reply:
Are you able to make this a boolean for those who don't need offline access?

Reply:
Added overloaded constructor with a boolean parameter to set this flag. Should help avoid breaking existing users projects too as this value will be set to `false` in the existing constructor